lib/src/rating/rating.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="rating-container" [ngClass]="className">
+    <div class="star-rating" [class.readonly]="readOnly" role="radiogroup" aria-labelledby="radioGroup">
```

Comment:
the `aria-labelledby` should be passed by the user if necessary, not hardcoded

Reply:
Moved it to the appropriate span element. It doesn't belong here.